test/lit/exec/table64.wast

```diff
@@ -0,0 +1,53 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --output=fuzz-exec and should not be edited.
+
+;; RUN: wasm-opt %s -all --fuzz-exec -q -o /dev/null 2>&1 | filecheck %s
+
+(module
+ (type $i32 (func (result i32)))
+
+ (table $table i64 10 funcref)
+ (elem (i64.const 0) $i32)
+
+ (func $i32 (result i32)
+  (i32.const 42)
+ )
+
+ ;; CHECK:      [fuzz-exec] calling call
+ ;; CHECK-NEXT: [fuzz-exec] note result: call => 42
+ (func $call (export "call") (result i32)
+  ;; This call succeeds, and calls $i32 which returns 42.
+  (call_indirect (type $i32)
+   (i64.const 0)
+  )
+ )
+
+ ;; CHECK:      [fuzz-exec] calling oob
+ ;; CHECK-NEXT: [trap callTable overflow]
+ (func $oob (export "oob") (result i32)
+  ;; This call traps on oob.
+  (call_indirect (type $i32)
+   (i64.const 999)
+  )
+ )
+
+ ;; CHECK:      [fuzz-exec] calling null
+ ;; CHECK-NEXT: [trap uninitialized table element]
+ (func $null (export "null") (result i32)
+  ;; This call traps on null
+  (call_indirect (type $i32)
+   (i64.const 1)
```

Comment:
Out of interest, is there a reason we use 1-space indentation binaryen?

Reply:
Sometimes we use 2-space indentation! IIRC, our printer uses 1-space indentation because very deep expression trees are common and the indentation levels would get way too high way too fast otherwise. Sometimes we write our tests with 1-space indentation to match the printer.

Reply:
Do we have such excessively deep expression trees in the test code?   Should the default be 2 (which we can then use in all the checked in tests) with an option to lower it when debugging "excessively deep expression trees" (or I guess just using flat syntax might make more sense for those cases?)